src/main/webapp/creatives/maps/SimidMap.js

```diff
@@ -0,0 +1,241 @@
+const DEFAULT_MAP_LAT = 37.422004;
```

Comment:
The rest of the js files weve been working on used snake_case_naming for the file name, so I would switch to that.

I also think we can change it from SimidMap, since thats pretty similar to SimidMapCreative, which is the main creative.  Something like GoogleMapsClient maybe? Anything that makes it clear how this is different from SimidMapCreative really would work

Reply:
Do you think renaming the object in the Simid Map creative would be a good idea as well?

Reply:
Yeah probably makes sense to keep it so that they match